src/routes/(app)/checkout/+page.svelte

```diff
@@ -189,7 +189,10 @@
 							name="shipping.firstName"
 							autocomplete="given-name"
 							required={data.roleId !== POS_ROLE_ID}
-							value={data.personalInfoConnected?.firstName ?? ''}
+							value={data.personalInfoConnected?.firstName ??
+							(data.roleId === POS_ROLE_ID && data.shopInformation?.businessName)
+								? data.shopInformation?.businessName
+								: ''}
```

Comment:
The priority is wrong, it acts like this because of missing parenthesis:

```suggestion
if (data.personalInfoConnected?.firstName ?? (data.roleId === POS_ROLE_ID && data.shopInformation?.businessName)) {
	return data.roleId === POS_ROLE_ID && data.shopInformation?.businessName;
} else {
	return '';
	}
```

It means that `personalInfoConnected` is not used anymore, even for non-POS users.

Same for others